src/addagent/manage_agents.c

```diff
@@ -98,14 +102,32 @@ int add_agent(int json_output)
     c_ip.ip = NULL;
 
     char *id_exist = NULL;
-    int force_antiquity = INT_MAX;
+    int disconnected_time = OS_INVALID;
```

Comment:
I can't remove the `disconnected_time` and `after_registration_time` variables because they represent the int value of the environment variables, but I removed one **if** by setting ` authd_force_options.enabled = true; `

Reply:
I think you can directly assign and use the **int** from **authd_force_options.after_registration_time** and **authd_force_options.disconnected_time**.

Reply:
You are right !